mitmproxy/script/concurrent.py

```diff
@@ -17,9 +17,11 @@ def concurrent(fn):
             "Concurrent decorator not supported for '%s' method." % fn.__name__
         )
 
-    def _concurrent(obj):
+    def _concurrent(*args):
+        obj = args[0] if len(args) == 1 else args[1]
```

Comment:
This won't be obvious if we look at it in a year, let's add a comment explaining what we are doing here. Also, we can simplify to `args[-1]` 😃

Reply:
nice, thank you for the tips 😄